docs/site/src/components/ToolbarButtons.js

```diff
@@ -0,0 +1,101 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import IconButton from 'material-ui/IconButton';
+import PlayCircleOutlineIcon from 'material-ui/svg-icons/play-circle-outline';
+import { camelCase, kebabCase } from 'docs/site/src/utils/helpers';
+import ApiMenu from './ApiMenu';
+import { apiMenus, componentMap, demoPaths } from './apiMenuData.js';
+
+const ToolbarButtons = (props, context) => {
```

Comment:
That would be good. 

We're starting to make our own style guide! (Maybe we add these little things to a wiki page, and it can evolve over time.)